json-resume-builder/src/components/Education.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+import resumeData from "../resume.json";
+
+export default class Education extends Component {
```

Comment:
some of these are super simple. have you considered using functional components?